src/stcal/jump/twopoint_difference.py

```diff
@@ -68,8 +72,11 @@ def find_crs(dataa, group_dq, read_noise, normal_rej_thresh,
     """
 
     # copy data and group DQ array
-    dataa = dataa.copy()
-    gdq = group_dq.copy()
+    if copy_arrs:
+        dataa = dataa.copy()
+        gdq = group_dq.copy()
+    else:
+        group_dq = gdq
```

Comment:
Is this expression backwards? The input to the function is `group_dq`, while `gdq` is used for subsequent processing below. So it should be `gdq = group_dq` (as above, but without the copy operation).

Reply:
yes, you're right